tests/integration/security/reachability_test.go

```diff
@@ -151,12 +151,21 @@ func TestReachability(t *testing.T) {
 					ExpectMTLS:    mtlsOnExpect,
 				},
 				{
-					ConfigFile:             "global-plaintext.yaml",
-					Namespace:              systemNM,
-					Include:                Always,
-					ExpectSuccess:          Always,
-					ExpectMTLS:             Never,
-					SkippedForMulticluster: true,
+					ConfigFile: "global-plaintext.yaml",
+					Namespace:  systemNM,
+					Include: func(src echo.Instance, opts echo.CallOptions) bool {
+						if apps.IsNaked(opts.Target) && len(apps.Naked.Clusters().ByNetwork()) > 1 {
```

Comment:
shouldn't we still include them? just not ExpectDestinations?

The only case we would want to exclude is when all naked pods are in other networks, and in that case we may want to actually include them but have ExpectSuccess=false?

Reply:
Adjusted this and ExpectMtls